test-configs.yaml

```diff
@@ -1635,16 +1675,56 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6qp-sabresd
```

Comment:
Baseline OK: https://staging.kernelci.org/test/plan/id/5efdb091532816a47348b90a/
Upstream in LAVA: https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/imx6qp-sabresd.jinja2